app/css/app.css

```diff
@@ -1,52 +1,20 @@
+/** General styles **/
 html {
+  height: 100%;
+
   font: normal 10px/10px Arial;
 }
 
 body {
+  height: 100%;
   margin: 0;
   padding: 0;
+
+  background: #FFF;
```

Comment:
Regarding case: definitely, lower case is much better!
Regarding aliases: honestly I'm against ```white``` since it's very limited set, so some colors will be in that notation, some will be definitely in a hex + it's much easier to copy-paste to visual editors since they all understand hex notation, but not human-readable aliases.